### PR TITLE
Goodbye --nosudo

### PR DESCRIPTION
**Pull request by atreyasha**
### Checklist

- [x] Search issues/pull-requests to ensure no duplicates
- [x] Update cram unit tests
- [x] Update `zsh`, `fish` and `bash` shell-completions
- [x] Update mandoc
- [x] Update usage function and readme
- [x] Update locales and add new translations

### Description

This PR addresses part of #138 by removing `--nosudo` related logic. ~Will hold merge until the other part of #138 is complete, ie. fixing the configuration file workflow.~

### CODE REVIEW

**Comment by atreyasha** [comment]
Just a quick question in regards to this modification. With the current diff, I noticed that now everything will require a `sudo` command for `downgrade`. So for example someone searching for help would input `downgrade --help` and the error message `downgrade must be run as root` will be brought up.

We could modify the parser to make things such that only the help option would be parsed without root permissions, but this would lead to a more complex diff. WDYT?

**Comment by atreyasha** [comment]
Or perhaps a slightly repetitive option would be to add the following before calling `cli`:

```shell
if (($(id -u))); then
    {
      gettext "downgrade must be run as root"
      echo
    } >&2

    for arg; do
      if [ "$arg" == "--help" ] || [ "$arg" == "-h" ]; then
        usage
      fi
    done

    exit 1
fi
```

**Comment by atreyasha** [comment]
Update on the last two comments: I moved the check for `-h|--help` earlier so that they are checked first before checking for root permissions. I found this better because also `pacman` does it this way, ie. it only requires root permissions when transacting package updates etc., but not for seeing the help script.

File: downgrade
Comment (pbrisbin) [inline downgrade:109]:
```suggestion
      printf "IgnorePkg = %s\\n" "$pkg" >> "$PACMAN_CONF"
```

I don't feel strongly, but this would work now since the shell itself has privileges now.

File: downgrade
Comment (pbrisbin) [inline downgrade:475]: Previous behavior, but I don't think `-h|--help` should exit unsuccessfully. The program did succeed in printing usage as asked.

Printing usage in response to an invalid argument should be non-zero, which is I think why this got this way by mistake.

File: downgrade
Comment (pbrisbin) [inline downgrade:480]:
```suggestion
  if ((UID)); then
```

Avoids spawning another process, and just a bit aesthetically nicer, IMO.

**Review by pbrisbin — APPROVED** [review]
Nice. Left some suggestions, but nothing blocking.

File: downgrade
Comment (atreyasha) [reply downgrade]: Makes sense. Added all of the suggestions and cleaned the commit history.

File: downgrade
Comment (atreyasha) [reply downgrade]: Ah yes, changed this.

File: downgrade
Comment (atreyasha) [reply downgrade]: Will change.

File: downgrade
Comment (atreyasha) [reply downgrade]: PS: Seems like this is a new GitHub feature that every new commit requires a new approval. Either that or I am somehow dismissing your reviews, which I don't know how haha.

In any case, I willl re-request your approval once I am done with the last step ie. locales.

Comment (pbrisbin) [reply downgrade]: That's my fault,



I must not have hits **Save** when I un-checked this earlier, it was checked.

I made sure to hit Save now, so we should be good going forward. If not, there's a bug in GitHub here somewhere.

**Comment by atreyasha** [comment]
FYI, replaced the `$` symbols in `README.md` to `#`; which AFAIK indicate root privileges.

**Comment by pbrisbin** [comment]
Makes total sense. Good catch!

**Review by pbrisbin — APPROVED** [review]
Awesome! Major bump I assume?

**Comment by atreyasha** [comment]
Nice :smile: I would suggest a minor bump for now and a major one after we complete the configuration file. WDYT?

**Pull request merged**
